src/Commands/PrepShooter.cpp

```diff
@@ -0,0 +1,29 @@
+#include "PrepShooter.h"
+#include "SetLifter.h"
+#include "ShooterControl.h"
+
+PrepShooter::PrepShooter(double mSpeed)
+{
+	speed = mSpeed;
```

Comment:
Exchange `speed` and `mSpeed`. The `m` prefix is used to indicate a member variable. Here, `mSpeed` is the parameter.